frontend/yarn.lock

```diff
@@ -2289,14 +2289,22 @@
   dependencies:
     "@math.gl/core" "3.4.2"
 
-"@math.gl/web-mercator@^3.1.3", "@math.gl/web-mercator@^3.3.0":
+"@math.gl/web-mercator@^3.3.0":
   version "3.4.2"
   resolved "https://registry.yarnpkg.com/@math.gl/web-mercator/-/web-mercator-3.4.2.tgz#72c7c7e698bcf8d1d1d60c0ec4c5a4698c9b960e"
   integrity sha512-Az/WI8vxbqnrTEcYgqDQ3CgCRoFA2a4XT9mkjVrT7iIlfrUF5lrIXcmpljjKvoFNBldKrng7hFSeHHM2ghgSrg==
   dependencies:
     "@babel/runtime" "^7.12.0"
     gl-matrix "^3.0.0"
 
+"@math.gl/web-mercator@^3.5.5":
+  version "3.6.3"
```

Comment:
This is a little curious to me. Above, it says ^3.3.0 -> 3.4.2 and ^3.5.5 -> 3.6.3. Shouldn't they both just point to 3.6.3?

This will reduce bundle size.